package.json

```diff
@@ -52,6 +52,7 @@
     "@typescript-eslint/parser": "^8.33.0",
     "@valora/prettier-config": "^0.0.1",
     "bignumber.js": "^9.2.1",
+    "bottleneck": "^2.19.5",
```

Comment:
nit: I see this lib hasn't been updated in 6 years, maybe we can switch with another one?
quick search yielded https://github.com/jhurliman/node-rate-limiter